### ISSUE TITLE
TypeError: mdpreview.mainModule.copyHtml is not a function

### ISSUE THREAD

**Issue by t-knapp** [travs/markdown-pdf#184]
### Prerequisites
Installed Markdown Preview and Markdown PDF Package.

### Issue description
Cannot create PDF file.

### Steps to reproduce
**Expected behavior:** Generate PDF file.
**Actual behavior:** Exception thrown.

### Input
On click on "Packages -> Markdown to PDF -> Convert" error pop-up is shown.
`markdown-pdf: TypeError: mdpreview.mainModule.copyHtml is not a function`

### Versions

Windows 7 64 Bit
>atom --version

Atom    : 1.24.1
Electron: 1.6.16
Chrome  : 56.0.2924.87
Node    : 7.4.0

**Comment by mensch** [comment]
I have the same issue on macOS High Sierra 10.13.3. Output from `atom --version`:

Atom    : 1.25.0
Electron: 1.7.11
Chrome  : 58.0.3029.110
Node    : 7.9.0

This was an issue in previous Atom builds as well.

**Comment by mensch** [comment]
I think I've found the issue (in my case anyway). I believe markdown-pdf tries to parse the user's stylesheet at first and when it encounters an error it switches from mpdf to fallback mode which appears not to be working anymore.

I was able to prevent markdown-pdf from switching to fallback mode by removing some code from my `styles.less`. An unresolvable import of `ui-variables` appears to have been the reason of markdown-pdf switching to legacy mode.

This is the error: `ui-variables.less' wasn't found. Tried - ui-variables.less,ui-variables.less`
Removing the import from `styles.less` allows markdown-pdf to create PDF.
